### ISSUE TITLE
Merge port request for "Ambitions and Objectives" PR

### ISSUE THREAD

**Issue by KrulPax** [ScorpioStation/ScorpioStation#292]
PR in question: https://github.com/Skyrat-SS13/Skyrat13/pull/2266

I believe this system would be amazing for our server, with the ability for traitors to access a special window in-game to pick their own objectives for the round without having to ahelp it. 

I believe this PR also adds a random chance to roll a "customizable traitor" which is a traitor whose sole "objective" is to set themselves a custom objective(s), and I feel that it'd be a cool addition if we could have like, 25% of a chance to roll this as a traitor, as long as there are admins online to oversee the type of objectives you'd be choosing. If that's not the case, however (PRs are so confusing!), I believe that it would be a good system on Scorpio as well.

**Comment by Kiyahitayika** [comment]
Nuuu~ Pax, this is an Issue! :D  Nah, it's fine that you did it as PR x3

Looks like Rohesie on Skyrat did excellent work, Skyrat uses the AGPL license, and it looks like we share a similar file structure - woohoo! If @blinkdog is okay with this concept, I can work on porting the code.

**Comment by blinkdog** [comment]
Yeah, that's looks pretty cool! I'd be in favor of merging it when we port it.

**Comment by Kiyahitayika** [comment]
Didn't mean to duplicate this. Couldn't find it for some reason.

Custom Antag Objectives.
